proto/lib/extras.go

```diff
@@ -449,7 +449,7 @@ func ImportRoleFromDB(t int, l int) (*Role, error) {
 		}
 		ret = NewRoleDefault(RoleType(t))
 	case RoleType_MEMBER:
-		if l < 0 || l > 0xffff {
+		if l < -0xffff || l > 0xffff {
```

Comment:
This change affects servers only, but needs to be deployed ahead of the v0.1.2 client release.